content/deploy/restore/keycloak.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Restore Keycloak'
+date: '2024-01-08T13:26:23-05:00'
+categories: ["how-to"]
+description: How to restore a Keycloak backup on Rhize
+weight: 300
+menu:
+  main:
+    parent: restore
+    identifier:
+---
+
+This guide shows you how to restore Keycloak in your Rhize environment.
+
+| Restoring Keycloak to an existing running instance will involve downtime
+
+Expected downtime for an existing running instance depends on the cluster performance sizing, backup size and bandwidth speed to the kubernetes cluster. Downtime is typically under a minute.
```

Comment:
> Expected downtime for an existing running instance depends on the cluster performance sizing, backup size and bandwidth speed to the Kubernetes cluster. Downtime is typically under a minute.

I can't quite understand what this means; I think my brain is especally tripping on "cluster performance sizing"―let's try a rewrite:

> For an existing running instance, the downtime needed to restore Keycloak depends on the size of the backup, the size and performance of the cluster, the network constraints between the cluster and the backup device.


Alternatively maybe this is two much info and we could just say:

> Restoring Keycloak involves downtime.

> Typically, this downtime lasts less than a minute. The exact duration needed depends on network constraints, backup size, and the performance of the Kubernetes cluster.

Are these accurate and suitable?